java/mapping/types.js

```diff
@@ -107,7 +107,9 @@
         constructor: (value) => `AccessibilityGuidelinesVersion.${value}`
     },
     "Location": {
-        constructor: (value) => `new Location(${value.x}, ${value.y})`
+        constructor: (value) => `new Location(${value.x}, ${value.y})`,
+        name: () => `Location`,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
        name: () => "Location",
```